style.css

```diff
@@ -0,0 +1,349 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
Use a decent font style for the project.
You can see from Google Fonts

Reply:
Done